packages/telestion-client-common/src/components/pages/not-found-page/not-found-message.tsx

```diff
@@ -4,6 +4,24 @@ import { Heading, IllustratedMessage, Content } from '@adobe/react-spectrum';
 
 import NotFound from '@spectrum-icons/illustrations/NotFound';
 
+/**
+ * Renders an illustrated message from React Spectrum
+ * containing a 404 page not found heading and a button to return
+ * to the dashboard page if the some is authenticated.
+ * Otherwise it returns to the login page.
```

Comment:
```suggestion
 * Renders an illustrated message from React Spectrum
 * containing a _404 page not found_ heading and a button to return
 * to the dashboard page, if the user is logged in.
 * Otherwise, redirects to the login page.
```